src/app/toolbar/toolbar.component.html

```diff
@@ -12,7 +12,11 @@
     <button aria-label="Info Dialog" class="btn btn-primary" (click)="openInfoDialog()">
       <i class="bi bi-info-circle"></i>
     </button>
+    <button aria-label="Chat history" class="btn btn-primary" (click)="toggleSavePanel()">
```

Comment:
another thing please also add an aria-label to every button you added which does not have a text itself as it is required for people which do not use/can use keyboards and use voice instead. Someone asked for this feature some time ago.